DependencyInjection/FOSUserExtension.php

```diff
@@ -71,6 +71,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load(sprintf('%s.xml', $basename));
         }
 
+        if ($config['use_authentication_listener']) {
+            $container->removeDefinition('fos_user.listener.authentication');
```

Comment:
Listener will be disabled when the "use_authentication_listener" is TRUE? Is that correct?